src/box/session.cc

```diff
@@ -375,3 +377,71 @@ generic_session_sync(struct session *session)
 	(void) session;
 	return 0;
 }
+
+API_EXPORT int
+box_session_id()
+{
+	return current_session()->id;
+}
+
+API_EXPORT bool
+box_session_exists(uint64_t sid)
+{
+	return (session_find(sid) != NULL);
+}
+
+API_EXPORT const char *
+box_session_peer(uint64_t sid)
+{
+	struct session *session = session_find(sid);
+	if (session == NULL)
+		session = current_session();
+	if (session == NULL)
+		return NULL;
+
+	int fd = session_fd(session);
+	if (fd < 0) {
+		return NULL;
+	}
+
+	struct sockaddr_storage addr;
+	socklen_t addrlen = sizeof(addr);
+	if (sio_getpeername(fd, (struct sockaddr *)&addr, &addrlen) < 0)
+		return NULL;
+
+	return sio_strfaddr((struct sockaddr *)&addr, addrlen);
```

Comment:
`sio_strfaddr()` returns a string allocated on the internal so called "static allocator". It is not documented, and is not safe to use in userspace code. I think it makes sense to make the user pass his own buffer. Just like `inet_ntop()` does.